tutorials/quantum_oracle/quantum_oracle_tutorial.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
<p>Maybe before "References" you might add mathematical analysis, if it's not too heavy. </p><p> </p><p>Also, Eden's example tutorial includes "All together" part of the code, and Qmod implementation (you can use write "write_qmod" function and just place it's output in markdown cell).</p><p> </p><p>Also consider adding executing the oracle and showing that the results are consist with our first intention (could be done from IDE "show" "execute" or directly from the SDK, for your choice). </p><p> </p><p>Check this for references: <a href="https://github.com/Classiq/classiq-library/blob/main/tutorials/phase_kickback/phase_kickback.ipynb" target="_blank">https://github.com/Classiq/classiq-library/blob/main/tutorials/phase_kickback/phase_kickback.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>